apps/tenant-management-webapp/src/app/store/configuration/sagas.ts

```diff
@@ -131,25 +140,38 @@ export function* fetchConfigurations(action: FetchConfigurationsAction): SagaIte
   );
 
   const token: string = yield call(getAccessToken);
-  const urls = getFetchUrls(action.services, configBaseUrl);
-  if (configBaseUrl && token && urls.length > 0) {
+
+  const urlsData = getFetchUrls(action.services, configBaseUrl);
+  if (configBaseUrl && token && urlsData.length > 0) {
     try {
       const configs = yield all(
-        urls.map((url) => call(axios.get, url, { headers: { Authorization: `Bearer ${token}` } }))
+        urlsData.map((urlData) => {
+          if (urlData.namespace) {
+            return call(fetchConfigurationNamespace, urlData, token);
+          } else {
+            return call(axios.get, urlData.url, { headers: { Authorization: `Bearer ${token}` } });
+          }
+        })
       );
+
       const { coreConfigDefinitions, tenantConfigDefinitions } = yield select(
         (state: RootState) => state.configuration
       );
       const definitions = { ...tenantConfigDefinitions?.configuration, ...coreConfigDefinitions?.configuration };
       yield put(
         getConfigurationsSuccess(
-          configs.map((c) => {
-            const key = `${c.data.namespace}:${c.data.name}`;
-            if (definitions[key] && definitions[key]?.configurationSchema?.description) {
-              c.data['description'] = definitions[key]?.configurationSchema?.description;
-            }
-            return c.data;
-          })
+          configs
+            .map((c) => {
+              if (Array.isArray(c)) {
+                return c;
+              }
+              const key = `${c.data.namespace}:${c.data.name}`;
```

Comment:
I'm a bit lost here: don't we have to worry about the name===null here?  Shouldn't this use the (new) toServiceKey() function?